### ISSUE TITLE
Birds of Steel

### ISSUE THREAD

**Issue by Triticum0** [RPCS3/rpcs3#13810]
### Quick summary

flying in the clouds causes graphical issues

### Details

The game seems to have these bars obstructing the front screen glass of the planes when viewed in the first-person camera angle when the plane is flying into clouds.

RPSC3: (Attack on Lexington)
![NPJB90501_screenshot_2023_05_05_21_55_50](https://user-images.githubusercontent.com/71127352/236569914-ac10d62e-5a6b-4618-831c-2ad28ec89b3f.png)

Note: I am not sure if this occurs on hardware as I have not tested it on the ps3

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11410154/RPCS3.log.gz)

### Attach capture files for visual issues

https://drive.google.com/file/d/1EQ6sjP3sM9f9Ll8AH7eSnVLNu59QjU4P/view?usp=sharing

### System configuration

RPCS3 v0.0.27-14962-b00b8ae7 Alpha | master | FW 4.88 | 5.15.106-1-MANJARO
AMD Ryzen 5 2600 | 12 Threads | 15.95 GiB RAM | AVX+ | FMA3
GPU: NVIDIA GeForce RTX 3060 Ti (530.41)

### Other details

I have not tested other configuration than the one tested in the log

**Comment by Triticum0** [comment]
Retested and there no change. Using other buffers + Cpu Blits
build (v0.0.30-15912-bc141831)

**Comment by digant73** [comment]
is that issue still present?

**Comment by Triticum0** [comment]
Yes but never test on hardware so don't know if it a bug or not but it did look like to me so I posted it.

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/6e53bb4f-4d43-42d0-825e-c8c92c51b8e0" />

Also found out that changing the SPU XFloat Accuracy  to Acccurate XFloat breaks the lighting

<img width="1280" height="720" alt="Image" src="https://github.com/user-attachments/assets/0417be4b-8c37-4d2e-b626-e23456d2991c" />

https://youtu.be/bYjByl8z-ik?t=57

**Comment by Triticum0** [comment]
Still an issue v0.0.40-192423
